public/donate.html

```
@@ -1,98 +1,698 @@
+    <!-- LEFT — DONATION FORM -->
+    <div class="lg:col-span-2 rounded-3xl border border-gray-200 dark:border-gray-700 bg-white dark:bg-gray-800 shadow-xl p-6 sm:p-8 lg:sticky lg:top-6">
+      <h2 class="text-2xl font-bold text-gray-900 dark:text-gray-100 mb-6">Make a Donation</h2>
+
+      <form id="donate-form" novalidate>
+
+        <!-- Donation Type -->
+        <fieldset class="mb-6">
+          <legend class="block text-sm font-semibold text-gray-700 dark:text-gray-300 mb-2">Donation Type</legend>
+          <div class="flex rounded-xl overflow-hidden border border-gray-300 dark:border-gray-600" role="group">
+            <label class="flex-1 cursor-pointer focus-within:ring-2 focus-within:ring-orange-500 focus-within:ring-inset">
+              <input type="radio" name="donation_type" value="one-time" class="sr-only" checked
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)